tools/run_tests/xds_k8s_test_driver/framework/infrastructure/gcp/compute.py

```diff
@@ -579,6 +586,18 @@ def _operation_status_done(operation):
     def _execute(  # pylint: disable=arguments-differ
         self, request, *, timeout_sec=_WAIT_FOR_OPERATION_SEC
     ):
+        if self.gfe_debug_header != DISABLE_DEBUG_HEADER_VALUE:
+
+            def _log_debug_header(resp):
+                if DEBUG_HEADER_IN_RESPONSE in resp:
+                    logger.info(
+                        "Received debug headers: %s",
+                        resp[DEBUG_HEADER_IN_RESPONSE],
+                    )
+
+            logger.info("Adding debug headers for method: %s", request.methodId)
+            request.headers[DEBUG_HEADER_KEY] = self.gfe_debug_header
+            request.response_callbacks.append(_log_debug_header)
```

Comment:
there's `request.add_response_callback` method

Reply:
oh, sorry missed it.